ci/prow/templates/prow_plugins.yaml

```diff
@@ -51,6 +51,19 @@ project_config:
           # TODO(grantr): replace with a new team eventing-contrib-project-maintainers
           repo_maintainers_team_id: 3012514 # knative-milestone-maintainers
 
+# Settings for the "label" plugin.
+label:
+  additional_labels:
+  - proposal/0.11
```

Comment:
Can we add a comment here saying "replace with proposal/* once https://github.com/kubernetes/test-infra/issues/15628 is solved"?

Reply:
I thought about that, but it can be that 15628 could be closed as WAI. In this case, there's nothing to do here.